### ISSUE TITLE
cache/stats files

### ISSUE THREAD

**Issue by szepeviktor** [szepeviktor/fix-w3tc#265]
I have `"stats.enabled": "0"` in master.json

And many tiny files in wp-content/cache/stats
```
./a57
./a57/ed6
./a57/ed6/a57ed60071e9972856374b21a8c3e6fb.php
./cf5
./cf5/cce
./cf5/cce/cf5cce3319de4354323df988181a2153.php
# etc.
```
Content
```
00000000  a6 d5 98 58 3c 3f 70 68  70 20 65 78 69 74 3b 20  |...X<?php exit; |
00000010  3f 3e 30                                          |?>0|
00000013
```

Is it possible to avoid creating these files?

**Comment by amiga-500** [comment]
I will check into this.

**Comment by amiga-500** [comment]
@szepeviktor the issue seems to be that it  makes a set of empty statistic files for potential future use (for pro license users who enable it) even when the statistics widget is disabled (ie. `stats.enabled=0`).  When enabled these stats appear in the bottom-right area of the dashboard and only shows caching/minify byte read/write info -- nothing important.   Since these files are never used (for the free version) it appears to be a coding mistake especially considering it will still make these unused files when it is disabled for license users.  

I've PR'ed a fix #269
